### PR TITLE
add package py-lmfit

### PR DESCRIPTION
**Pull request by healther**
* needed to deactivate testing in openblas due to spec.cppflags being undefined

Should I open an issue? I could also directly fix the openblas error, if someone can tell me how to do it.

### CODE REVIEW

File: var/spack/repos/builtin/packages/openblas/package.py [hunk 1]
```diff
@@ -129,26 +129,28 @@ def install_targets(self):
         ]
         return make_args + self.make_defs
 
-    @run_after('install')
-    def check_install(self):
-        spec = self.spec
```
Comment (davydden) [inline var/spack/repos/builtin/packages/openblas/package.py]: please don't  comment out the whole thing. If the issue is with spec.cppflags` being undefined, then at least test it at the beginning of this function and return if that is the case.

Comment (davydden) [inline var/spack/repos/builtin/packages/openblas/package.py]: I looked closer, this is a regression. Someone needs to simply fix it. That was not a problem a day ago, but right now I see the issue as well.

Comment (davydden) [inline var/spack/repos/builtin/packages/openblas/package.py]: either way, do NOT comment out this method, that's hardly a solution to the problem.

**Comment by adamjstewart** [comment]
The Python package looks good. If you undo the OpenBLAS changes, I think this is good to merge. @alalazo @davydden can you take a look at OpenBLAS and see why it is failing? It should be an easy fix. Also, we should only run these tests when `--run-tests` is requested.

**Comment by healther** [comment]
It wasn't thought as a solution, rather I wanted to know how I should proceed.

Undid the ```openblas``` change

**Pull request merged**